environments/association/argocd-app-templates/appsetup-int.yaml

```diff
@@ -0,0 +1,38 @@
+###############################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################
+
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: semantic-dec-registry
+spec:
+  destination:
+    namespace: product-semantics
+    server: 'https://kubernetes.default.svc'
+  source:
+    path: charts/registry
+    repoURL: 'https://github.com/eclipse-tractusx/sldt-digital-twin-registry.git'
+    targetRevision: main
```

Comment:
This is pointing to main and sync with main. Does it make sense to introduce chart.yml etc. to define helm versions? 
Otherwise it is not possible to define a released helm version.

Reply:
Yes I will change this to tagged versions instead of main branch.

Reply:
i think we need to change the way it is. 
Because this script runs one time to create the app. 
If we keep it like this (main), the newest helm which is defined will be taken. 
If you change to a tag - How it looks when we want to deploy a newer helm version?